cli/productutil.py

```diff
@@ -28,7 +28,7 @@
 )
 from product.util import merge_products, ComponentDescriptorResolver
 from product.scanning import ProcessingMode
-from protecode.util import upload_images
+from protecode.util import process_images
```

Comment:
yeah, why not. this commit would also be a candidate for being pushed to master branch already, as it does not add to the PR's semantical focus imo

why do you think 'process' is a better name than 'upload'? both fine with me. i just do not see the added semantics resulting from the change of names